dms-infrastructure/src/main/kotlin/team/aliens/dms/thirdparty/parser/ExcelAdapter.kt

```diff
@@ -155,27 +247,56 @@ class ExcelAdapter : ParseFilePort, WriteFilePort {
     }
 
     private fun insertDatasAtRow(
-        headerRow: XSSFRow,
-        attributes: List<String?>,
-        style: XSSFCellStyle
+        row: Row,
+        datas: List<String?>,
+        style: CellStyle,
     ) {
-        attributes.forEachIndexed { j, text ->
-            val cell = headerRow.createCell(j)
-            cell.setCellValue(text)
+        datas.forEachIndexed { i, data ->
+            val cell = row.createCell(i)
+            data?.toIntOrNull()?.toDouble()?.let {
+                cell.setCellValue(it)
+            } ?: cell.setCellValue(data)
             cell.cellStyle = style
         }
     }
 
-    private fun getHeaderCellStyle(workbook: XSSFWorkbook): XSSFCellStyle =
-        workbook.createCellStyle()
+    private fun insertDatasAtRow(
+        row: Row,
+        startIdx: Int,
```

Comment:
```suggestion
        startIdx: Int = 0,
```
으로 초기화해줘서 같은 함수 사용할 수 있을 것 같아요